etcd/fs/fs_test.go

```diff
@@ -39,6 +43,31 @@ const (
 	superblock = "/some/superblock" // TODO: generate randomly per test?
 )
 
+func TestMain(m *testing.M) {
```

Comment:
Sweet!

Can you add a small comment about what TestMain is, and also why we're calling ourself of course? Thanks!!